tools/libs/core.sh

```diff
@@ -51,7 +51,7 @@ link_pkglist_generic() {
 }
 
 run_apt_update() {
-    apt-get -q --allow-releaseinfo-change update
+    sudo -u "${BASE_USER}" apt-get -q --allow-releaseinfo-change update
```

Comment:
This isn't needed as you install it with `sudo`

Reply:
There seems to be some problem with understanding https://github.com/mainsail-crew/crowsnest/issues/141

It appears you have not tried building an image via custompios and docker.  It does not work with the scripts as is without my PR because the default user is root in docker.

Also, this file clearly has inconsistencies everywhere.  Some places you sudo as BASE_USER to invoke some calls some you do not, even though you should have.

Reply:
There is no inconsistency. You normally don't need to install as a user, as apt will install system-wide. So installation can be done without extra specifying an user. `sed` and `ln` also doesn't matter if they are executed as the user or with `sudo` so it isn't needed for that too.
If you now copy a file, it matters, if you use `-u` with `${BASE_USER}` or if you just use `sudo`, as you will have wrong permissions otherwise.
So the use of `sudo -u` and not, is consistent and `sudo -u` is only used for the most important parts